programs/mango-v3-reimbursement/src/instructions/change_group_authority.rs

```diff
@@ -0,0 +1,25 @@
+use anchor_lang::prelude::*;
+
+use crate::state::Group;
+
+#[derive(Accounts)]
+pub struct ChangeGroupAuthority<'info> {
+    #[account(
+        mut,
+        has_one = authority,
+        constraint = group.load()?.is_testing(),
```

Comment:
I'd not put in this constraint, we need to use it on a live group for easier setup